backend/src/v2/driver/driver.go

```diff
@@ -665,6 +665,33 @@ func extendPodSpecPatch(
 		podSpec.Volumes = append(podSpec.Volumes, ephemeralVolume)
 		podSpec.Containers[0].VolumeMounts = append(podSpec.Containers[0].VolumeMounts, ephemeralVolumeMount)
 	}
+
+	// EmptyDirMounts
+	for _, emptyDirVolumeSpec := range kubernetesExecutorConfig.GetEmptyDirMounts() {
+		var sizeLimitResource *k8sres.Quantity
```

Comment:
We could add some logging here to indicate the attempt to create and mount the EmptyDir volume. 
```
log.Infof("Attempting to create EmptyDir volume '%s' and mount it at '%s' with medium '%s'",
        emptyDirVolumeSpec.GetVolumeName(),
        emptyDirVolumeSpec.GetMountPath(),
        emptyDirVolumeSpec.GetMedium(),
    )
   ```
   Just a nice to have, no compulsion :)

Reply:
+1, gonna do this in a followup overall refactor of driver :)